mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/260 - character freeze on dialog up

Closes #260 

To test:
- Run Training/Stop Worms scenario
- Try to get camera in position so character is in corner (so they'll be visible while dialog is up)
- Proceed through scenario until message dialog pops up
- Make sure characters freeze while dialog is up
